examples/string-interpolation/Python.py

```diff
@@ -0,0 +1,92 @@
+'''
+STRING INTERPOLATION
+'''
+
+name = "John Doe"
+age = 20
+
+#These are different ways you can combine variables together
+
+
+#Concatenation (The one you are most familiar with)
+string = "Hello! My name is " + name + " and I am " + str(age) + " years old."
+print(string)
+
+#f-strings (Works only for Python 3.6+)
+# placing 'f' before initializing a string, denotes the start of an f-string
+string = f"Hello! My name is {name} and I am {age} years old."
+print(string)
+
+#Formatting (Works for all versions of Python)
+# {} denotes a placeholder for a variable
+string = "Hello! My name is {} and I am {} years old.".format(name, age) #Variables are put in order at which they appear
+string = "Hello! My name is {name} and I am {age} years old.".format(name=name, age=age) #Can also be written like this
```

Comment:
This might be a confusing example for newer programmers (especially the `name=name` and `age=age` portions).

Reply:
Would it be less confusing to use different placeholder names?

Reply:
That would probably be the best place to change things, yes.